docs/tutorials/analysis/2D/ring_background.ipynb

```diff
@@ -118,10 +118,10 @@
     "    \"lat\": source_pos.dec,\n",
     "    \"frame\": \"icrs\",\n",
     "}  # The WCS geometry - centered on MSH 15-52\n",
-    "config.datasets.geom.wcs.fov = {\"width\": \"3 deg\", \"height\": \"3 deg\"}\n",
+    "config.datasets.geom.wcs.width = {\"width\": \"3 deg\", \"height\": \"3 deg\"}\n",
     "config.datasets.geom.wcs.binsize = \"0.02 deg\"\n",
     "\n",
-    "# The FoV radius to use for cutouts\n",
+    "# The radius to use for cutouts\n",
```

Comment:
I don't know if this one is very clear. Should we use cutout size? 
Do we really need it?

Reply:
Maybe we don't really need it, and what we need is an additional glossary entry for the offset_max? For the moment I kept it here, with a (hopefully) clearer formulation